tests/analyzer/s7ident.zeek

```diff
@@ -0,0 +1,9 @@
+# @TEST-EXEC: zeek -C -r ${TRACES}/s7ident.pcap %INPUT
+# @TEST-EXEC: btest-diff cotp.log
+# @TEST-EXEC: btest-diff s7comm.log
+# @TEST-EXEC: btest-diff s7comm_plus.log
```

Comment:
This `s7comm_plus.log` file is not generated when running your branch against `s7ident.pcap`, at least not under Zeek 7.1.0, so this test fails. But even the current `main` branch doesn't generate that log file either, so I'm not sure why you're expecting it here?